IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_DoorTypeTurnstile/DocPropertySet.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_DoorTypeTurnstile" UniqueId="c5233710-61f4-8eb7-185b-60b03c52ca80" ApplicableType="IfcDoor/TURNSTILE" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="IsBidirectional_3PYJ6a4gXACx94JYuDo4no" />
+		<DocProperty xsi:nil="true" href="GateID_1vaewDvSn2zgrPRZgB_Hg9" />
+		<DocProperty xsi:nil="true" href="PassengerThroughput_0bZXuIvtrB3QZWBy_YgRO2" />
+		<DocProperty xsi:nil="true" href="TicketProcessingTime_17M9Cc_kz818DfWA4GZJTA" />
+		<DocProperty xsi:nil="true" href="TicketStuckRate_3yrQ19j3XCuR09jcP_sUQK" />
+		<DocProperty xsi:nil="true" href="TurnstileType_3yEcMPun2Du7wMts2JRaM" />
+		<DocProperty xsi:nil="true" href="NarrowChannelWidth_260oCTccX4deJpXc0XFb_" />
+		<DocProperty xsi:nil="true" href="WideChannelWidth_3SSA9HxpbBzeAzEvzUdRN" />
```

Comment:
I am assuming this is a kin to NominalWidth as a design parameter/product property rather than a geometrically derived  quantity correct?

Reply:
That is correct

Reply:
Resolved